Src/Support/GoogleApis.Auth.DotNet4.Tests/GoogleApis.Auth.PlatformServices.Tests_Net45.csproj

```diff
@@ -15,6 +15,7 @@
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
     <TargetFrameworkProfile />
+    <StartArguments>--wait</StartArguments>
```

Comment:
Yes, this is just what VS will use if you hit F5 with no further customization. It doesn't get compiled into the test so it doesn't affect behavior when using a test runner. It can also be overridden in VS (which will write a .csproj.user file).